CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0] - 2024-06-05
+
+### Changes
+
+- Fixed the session refresh loop in all the request interceptors that occurred when an API returned a 401 response despite a valid session. Interceptors now attempt to refresh the session a maximum of ten times before throwing an error. The retry limit is configurable via the `maxRetryAttemptsForSessionRefresh` option.
+
```

Comment:
check if you need a migration thing here based on our discussion

Reply:
We don't. Swift by default enforces labeled arguments. 
<img width="1111" alt="image" src="https://github.com/supertokens/supertokens-ios/assets/22813027/a31948ef-d3ed-4307-ac2d-03cfcca433ef">